app/models/config-model.js

```diff
@@ -259,7 +259,8 @@ function getThemesSupported( themeList ) {
 }
 
 try {
-    config[ 'version' ] = execSync( 'git describe --tags', { encoding: 'utf-8' } ).trim();
+    // need to be in the correct directory to run git describe --tags
+    config[ 'version' ] = execSync( 'cd ' + config[ 'home directory' ] + '; git describe --tags', { encoding: 'utf-8' } ).trim();
```

Comment:
There must be a better way to do this, no? The script should know it's own location, right? If so, please send PR to enketo/enketo-express for this change.

Reply:
Also use template literal https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
the script does not find out its own location, thats why we have to add this. Will use template literal.